packages/database/src/migrations/20210602065620-AddFijiCOVIDVaccinatedRegionOverlays-modifies-data.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getOverlayId = (doseNum, level) => `FJ_COVID_TRACKING_Dose_${doseNum}_${level}`;
+
+const createMapOverlay = (doseNum, aggregationEntityType, level, overlayLevelName) => ({
+  id: getOverlayId(doseNum, level),
+  name: `COVID-19 Vaccine Dose ${doseNum} (${overlayLevelName})`,
+  userGroup: 'COVID-19',
+  dataElementCode: doseNum === 1 ? 'COVIDVac4' : 'COVIDVac8',
+  measureBuilderConfig: {
+    aggregations: [
+      {
+        type: 'FINAL_EACH_DAY',
+      },
+    ],
+    entityAggregation: {
+      dataSourceEntityType: 'facility',
+      aggregationEntityType,
+    },
+  },
+  measureBuilder: 'sumAllPerOrgUnit',
```

Comment:
This migration is for: beyondessential/tupaia-backlog#2907
The overlay is shaded polygons at district/sub_district level. We want to get latest for each day for a facility, and aggregate (sum) to district/sub_district level